dev/io.openliberty.restfulWS.3.0.cdi.3.0_fat/publish/servers/io.openliberty.restfulWS.3.0.cdi.3.0.fat.lifecyclemismatch/bootstrap.properties

```diff
@@ -0,0 +1,5 @@
+com.ibm.ws.logging.trace.specification=*=info:\
+com.ibm.ws.jaxrs20.cdi.component.JaxRsFactoryImplicitBeanCDICustomizer=all
```

Comment:
holy crap this one caused problems, the test is looking through the trace file for System.out statements rather than the messages.log and by removing this I no longer generated a trace.log file.